examples/copula/R/centered_gaussian_copula_example.R

```diff
@@ -1,8 +1,10 @@
 library(cmdstanr)
 # setwd("~\helpful_stan_functions")
-fp <- file.path("./examples/copula/stan/mixed_copula_example.stan")
-mod <- cmdstan_model(fp, include_paths = c("./functions/copula",
-                                          "./functions/distribution"))
+fp <- file.path("./examples/copula/stan/centered_gaussian_copula_example.stan")
+mod <- cmdstan_model(fp, include_paths = c("./functions",
+                                           "./functions/copula",
+                                           "./functions/distribution",
+                                           "../functions"))
```

Comment:
the "../functions" line causes an error for me. Does your machine need this?

Reply:
Ah no it doesn't, I'm not sure why I had that. Will pull out